primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.inputtextarea.js

```diff
@@ -55,6 +55,9 @@ PrimeFaces.widget.InputTextarea = PrimeFaces.widget.DeferredWidget.extend({
         //autoComplete
         if(this.cfg.autoComplete) {
             this.setupAutoComplete();
+            if (!this.jq.hasClass('ui-state-disabled')) {
+                this.jq.data('primefaces-overlay-target', true);
+            }
```

Comment:
I think a better place for this is the last line of of the `setupAutoComplete` method.